package.json

```diff
@@ -69,6 +69,8 @@
     "karma-ng-html2js-preprocessor": "^1.0.0",
     "karma-webpack": "^4.0.0-rc.3",
     "loader-utils": "^2.0.0",
+    "lighthouse": "5.6.0",
+    "@lhci/cli": "0.3.12",
```

Comment:
Add `^` before the version number so that it gets updated automatically.

Reply:
Done.